spec.html

```diff
@@ -37417,67 +37419,121 @@ <h1>Runtime Semantics: QuoteJSONString ( _value_ )</h1>
           1. Set _product_ to the string-concatenation of _product_ and the code unit 0x0022 (QUOTATION MARK).
           1. Return _product_.
         </emu-alg>
+        <emu-table id="table-json-single-character-escapes" caption="JSON Single Character Escape Sequences">
+          <table>
+            <tbody>
+            <tr>
+              <th>
+                Code Unit Value
+              </th>
+              <th>
+                Unicode Character Name
+              </th>
+              <th>
+                Escape Sequence
+              </th>
+            </tr>
+            <tr>
+              <td>
+                `0x0008`
+              </td>
+              <td>
+                BACKSPACE
+              </td>
+              <td>
+                `\\b`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                `0x0009`
+              </td>
+              <td>
+                CHARACTER TABULATION
+              </td>
+              <td>
+                `\\t`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                `0x000A`
+              </td>
+              <td>
+                LINE FEED (LF)
+              </td>
+              <td>
+                `\\n`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                `0x000C`
+              </td>
+              <td>
+                FORM FEED (FF)
+              </td>
+              <td>
+                `\\f`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                `0x000D`
+              </td>
+              <td>
+                CARRIAGE RETURN (CR)
+              </td>
+              <td>
+                `\\r`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                `0x0022`
+              </td>
+              <td>
+                QUOTATION MARK
+              </td>
+              <td>
+                `\\"`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                `0x005C`
+              </td>
+              <td>
+                REVERSE SOLIDUS
+              </td>
+              <td>
+                `\\\\`
+              </td>
+            </tr>
+            </tbody>
+          </table>
+        </emu-table>
+      </emu-clause>
+
+      <emu-clause id="sec-unicodeescape" aoid="UnicodeEscape">
+        <h1>Runtime Semantics: UnicodeEscape ( _C_ )</h1>
+        <p>The abstract operation UnicodeEscape takes a code unit argument _C_ and represents it as a Unicode escape sequence.</p>
+        <emu-alg>
+          1. Let _n_ be the numeric value of _C_.
+          1. Return the string-concatenation of:
+            * the code unit 0x005C (REVERSE SOLIDUS)
+            * `"u"`
+            * the String representation of _n_, formatted as a four-digit lowercase hexadecimal number, padded to the left with zeroes if necessary
```

Comment:
In a step before this, I would assert that `C` is not greater than 65,535 (`0xFFFF`).